### PR TITLE
Refactor Game Class and remove game class spec

### PR DESCRIPTION
**Pull request by suzkiee**
## What does this change?

user.inputs 

## What does this fix?

Make user.inputs not case sensitive

## Is this a bug or a feature? 

bug fix 

## How was this tested?

We ran the game to double check.

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -55,13 +48,15 @@ def celebrate_winner(user, computer)
...
-    computer_setup = Setup.new(ships, @computer_board, :computer)
+    board = Board.new
+    computer_setup = Setup.new(ships, board, :computer)
     computer = computer_setup.run_setup
   end
 
   def complete_user_setup
     ships = create_ships
-    user_setup = Setup.new(ships, @user_board, :user)
+    board = Board.new
```
Comment (mkrumholz) [inline lib/game.rb:58]: actually really like having these in these methods.

File: lib/setup.rb [hunk 2]
```diff
@@ -45,13 +45,13 @@ def user_setup_intro(ships, board)
   def user_place_ship(ship, board)
     puts "\nEnter the coordinates for #{ship.name} (#{ship.length} spaces, format: A1 B1 ):"
     print "> "
-    user_input = gets.chomp
+    user_input = gets.chomp.upcase
```
Comment (mkrumholz) [inline lib/setup.rb:48]: Major improvement for usability

File: lib/game.rb [hunk 3]
```diff
@@ -1,12 +1,5 @@
 class Game
 
-  attr_reader :user_board,
```
[the remark this answers is not in the document]
Comment (suzkiee) [reply lib/game.rb:3]: yessssss

File: lib/game.rb (hunk 1 above)
Comment (suzkiee) [reply lib/game.rb:58]: our win this morning for sure!

**Pull request merged**